controllers/activemqartemis_reconciler.go

```diff
@@ -2622,6 +2630,13 @@ func UpdateBrokerPropertiesStatus(cr *brokerv1beta1.ActiveMQArtemis, client rtcl
 				Reason:  brokerv1beta1.ConfigAppliedConditionOutOfSyncReason,
 				Message: brokerv1beta1.ConfigAppliedConditionOutOfSyncMessage,
 			}
+		case inSyncApplyError:
+			condition = metav1.Condition{
+				Type:    brokerv1beta1.ConfigAppliedConditionType,
+				Status:  metav1.ConditionTrue,
```

Comment:
Should the `status` be false?

Reply:
that is the question. I think not, b/c the condition is about being in sync. but I think it could go either way... the conditions are  accumulative, so ready would be false b/c this condition is false. I can go either way on this... having a bla=bla property that has been ignored, from a CR status perspective, think it is minor.

Reply:
I think you are right. The genesis of the condition is the delay in a config mount projection, has the broker seen the projection update (with a mutable) config map. that can take up to 2mins. But I guess users don't care about that really, it is fine to be unknown, then true or false based on the actual content being valid rather than just being applied.
you win! thanks for the feedback.